client/hud_cl.lua

```diff
@@ -265,3 +265,18 @@ function StopAnim()
 		prop = 0
 	end
 end
+
+CreateThread(function()
+	while true do
+	Citizen.Wait(0)
+	
+		local playerPed = PlayerPedId()
+		local playerVeh = GetVehiclePedIsIn(playerPed, false)
+
+		if DoesEntityExist(playerVeh) then

```

Comment:
Couldn't we just do `DisplayRadar(DoesEntityExist(vehicle))`?
If it doesn't work, then it's fine.